parse/getast.go

```diff
@@ -345,6 +346,9 @@ func (fs *FileSet) getField(f *ast.Field) []gen.StructField {
 		if len(tags) == 2 && tags[1] == "extension" {
```

Comment:
It might make sense to refactor this to
```go
if len(tags) >= 2 {
    switch tags[1] {
    case "extension": ...
    case "flatten": ...
    }
}
```

Reply:
done